apps/docs/src/app/core/component-docs/resizable-card-layout/resizable-card-layout-docs-header/resizable-card-layout-docs-header.component.html

```diff
@@ -0,0 +1,11 @@
+<header>Resizable Card Layout 
+    <fd-info-label label="Experimental" title="development in-progress"></fd-info-label>
+</header>
+<description>
+    Resizable Card Layout enables users to define a personalized card layout by changing size of the card as well as position of the card.
```

Comment:
how can we change the position? is it through DnD or through the API?

Reply:
documented